rules/qunit.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  plugins: [
+    'qunit',
+  ],
+  rules: {
+    'qunit/assert-args': 'error',
+    'qunit/literal-compare-order': 'error',
+    'qunit/no-arrow-tests': 'error',
+    'qunit/no-assert-equal': 'off',
+    'qunit/no-assert-logical-expression': 'off',
+    'qunit/no-async-in-loops': 'error',
+    'qunit/no-async-test': 'error',
+    'qunit/no-commented-tests': 'error',
+    'qunit/no-compare-relation-boolean': 'error',
+    'qunit/no-conditional-assertions': 'error',
+    'qunit/no-early-return': 'off',
+    'qunit/no-global-assertions': 'off',
+    'qunit/no-global-expect': 'off',
+    'qunit/no-global-module-test': 'off',
```

Comment:
Should this be switched on? Not sure whether that would fire when we `import` `module` etc.?